asv_bench/benchmarks/groupby.py

```diff
@@ -369,23 +377,27 @@ class GroupByMethods(object):
     goal_time = 0.2
 
     param_names = ['dtype', 'method']
-    params = [['int', 'float'],
+    params = [['int', 'float', 'object'],
```

Comment:
I didn't end up removing any other tests after adding this here so you could argue there's some bloat with this. That said, my plan was to take care of #19733 first and then come back to clean up where possible. The reason for that is some of the existing object tests are wrapped up with `datetime` tests as well (see `class FirstLast`) so it would add some complexity to try and unwind that before completing the change mentioned

Reply:
thats fine